src/sqlfluff/dialects/dialect_mysql.py

```diff
@@ -1582,7 +1581,7 @@ class DropFunctionStatementSegment(BaseSegment):
     https://dev.mysql.com/doc/refman/8.0/en/drop-function-loadable.html
     """
 
-    type = "drop_function_ statement"
+    type = "drop_function_statement"
```

Comment:
Could we remove the whole segment definition from MySQL?

Reply:
Technically yes, but I liked the fact it linked to it's own definition (which is admittedly the same as what we've implemented in ANSI) in the docstring so I left it in there.

https://github.com/sqlfluff/sqlfluff/blob/0cfccf4a927d71dc2b7619345e962ebaa2893d98/src/sqlfluff/dialects/dialect_mysql.py#L1578-L1582